docs/Data/Bioinformatics_Pipelines/CNV_Pipeline.md

```diff
@@ -8,7 +8,7 @@ The copy number variation (CNV) pipeline uses either NGS or Affymetrix SNP 6.0 (
 * ABSOLUTE
 * DNAcopy
 
-The first set of CNV pipelines are built upon the ASCAT [[1]](https://www.pnas.org/content/107/39/16910) algorithm for both WGS and SNP6 data. ASCAT is able to generate Allele-specific Copy Number Segment data with integer copy number values, and the derived integer Gene-Level Copy Number. 1.) The WGS copy number analysis pipeline, [ascatNGS](https://github.com/cancerit/ascatNgs), is described in detail [here](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/#whole-genome-sequencing-variant-calling). 2.) The SNP6 copy number analysis pipeline, ASCAT2, is adopted from the [example ASCAT analysis](https://github.com/VanLoo-lab/ascat/blob/v2.5/ExampleData/ASCAT_examplePipeline.R). 3.) The SNP6 copy number analysis pipeline, ASCAT3, is an updated version of ASCAT2. The ASCAT3 analysis in TCGA was done by the [Vanloo lab](https://github.com/VanLoo-lab/ascat/tree/master/ReleasedData/TCGA_SNP6_hg38), and the GDC released a reformatted version of these calls. Both ASCAT2 and ASCAT3 generates data similar to ascatNGS.
+The first set of CNV pipelines are built upon the ASCAT [[1]](https://www.pnas.org/content/107/39/16910) algorithm for both WGS and SNP6 data. ASCAT is able to generate Allele-specific Copy Number Segment data with integer copy number values, and the derived integer Gene-Level Copy Number. 1.) The WGS copy number analysis pipeline, [ascatNGS](https://github.com/cancerit/ascatNgs) (__now deprecated at GDC__), is described in detail [here](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_WGS/). 2.) The SNP6 copy number analysis pipeline, ASCAT2, is adopted from the [example ASCAT analysis](https://github.com/VanLoo-lab/ascat/blob/v2.5/ExampleData/ASCAT_examplePipeline.R). 3.) The SNP6 copy number analysis pipeline, ASCAT3, is an updated version of ASCAT2. The ASCAT3 analysis in TCGA was done by the [Vanloo lab](https://github.com/VanLoo-lab/ascat/tree/master/ReleasedData/TCGA_SNP6_hg38), and the GDC released a reformatted version of these calls. Both ASCAT2 and ASCAT3 generates data similar to ascatNGS.
```

Comment:
I think "...now deprecated at the GDC..." sounds better. applies to other instances in the PR.